service/src/main/java/io/github/solomkinmv/glossary/service/images/ImageServiceImpl.java

```diff
@@ -20,15 +20,18 @@
 
 @Service
 @Slf4j
-public class ImageServiceImpl implements ImageService {
+class ImageServiceImpl implements ImageService {
 
+    private static final int IMAGE_WIDTH = 240;
```

Comment:
If there would be any necessity I will extract this to properties. Currently I'm not sure about this

_Sent from my Google Nexus 5 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_